mllm/backends/ascend/Register.cpp

```
@@ -0,0 +1,41 @@
+#include "mllm/mllm.hpp"
+#include "mllm/backends/ascend/AscendBackend.hpp"
+#include "mllm/backends/ascend/AscendDispatcher.hpp"
+#include "mllm/backends/ascend/memory/AscendMemoryManager.hpp"
+
+namespace mllm {
+
+void initAscendBackend() {
+  auto& ctx = Context::instance();
+
+  // 1. Create memory pool
+  size_t pool_size = 100 * 1024 * 1024;  // 100MB, can be adjusted as needed
```

Clang (14.0.6)

[error] variable 'pool_size' is not initialized

(cppcoreguidelines-init-variables,-warnings-as-errors)